### PR TITLE
[WIP] Add blog post for kubelet credential provider

### PR DESCRIPTION
**Pull request by adisky**
Signed-off-by: Aditi Sharma <adi.sky17@gmail.com>
Placeholder PR for Kubelet Credential Provider Feature Blog Post https://github.com/kubernetes/enhancements/issues/2133#issuecomment-1078672369

### CODE REVIEW

**Comment by valaparthvi** [comment]
Hi @adisky! I'm from the Comms team, and I wanted to check on the status of this blog. Reviews for feature blogs are supposed to start next week. Anything we can help you with?

**Comment by mickeyboxell** [comment]
@adisky could we plan to have this ship on 10-MAY-2022?

**Comment by mickeyboxell** [comment]
@adisky Could we change the title to 2022-05-09-kubelet-credential-provider.md

File: content/en/blog/_posts/2022-03-25-kubelet-credential-provider.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: blog
+title: 'Kubernetes 1.24: Kubelet Credential Provider Reaches Beta'
+date: 2022-03-25
```
Comment (mickeyboxell) [inline content/en/blog/_posts/2022-03-25-kubelet-credential-provider.md:4]:
```suggestion
date: 2022-05-09
```

Comment (sftim) [reply content/en/blog/_posts/2022-03-25-kubelet-credential-provider.md:4]: Definitely the 9th of May (a Monday)? I saw the 10th also mentioned.

Please update file paths to match the intended publication date.

**Comment by sftim** [comment]
It looks like this has been missed (there's no content to review and the target date has passed).

**Comment by nate-double-u** [comment]
/milestone clear
v1.24 has been released, but this post can come in after.

**Comment by sftim** [comment]
No article content.
/close



We'd be happy to review a separate article (in a new PR) that isn't linked to the v1.24 release comms.

**Pull request closed without merging**